health/micro-ui/web/micro-ui-internals/packages/modules/campaign-manager/src/components/MobileBezelFrame.js

```
@@ -0,0 +1,59 @@
+import React from "react";
+
+const MobileBezelFrame = ({ children }) => {
+  return (
+    <div className="mobile-bezel-outerWrapper" style={styles.outerWrapper}>
+      <div className="mobile-bezel-deviceWrapper" style={styles.deviceWrapper}>
+        <div className="mobile-bezel-camera" style={styles.camera}></div>
```

Biome (1.9.4)

[error] JSX elements without children should be marked as self-closing. In JSX, it is valid for any element to be self-closing.

Unsafe fix: Use a SelfClosingElement instead


(lint/style/useSelfClosingElements)